.github/workflows/main.yml

```
@@ -75,7 +75,7 @@ jobs:
       - name: Coverage
         run: |
           make coverage
-          sed -i "/^.*internal\/bindata\/cldr.*$/d" ./cover.out
+          sed -i "/^.*internal\/bindata.*$/d" ./cover.out
```

bindata is a folder. Ignore anything in it.
```suggestion
          sed -i "/^.*internal\/bindata\/.*$/d" ./cover.out
```